core/shared/src/main/scala/zio/Config.scala

```diff
@@ -290,6 +291,13 @@ object Config {
       case NonFatal(e) => Left(Config.Error.InvalidData(Chunk.empty, s"Expected a local time value, but found ${text}"))
     }
   }
+  case object LogLevelType extends Primitive[LogLevel] {
```

Comment:
We should not introduce additional primitives here.

Reply:
should it be just 

```scala
  def logLevel: Config[LogLevel] = Config.string.mapOrFail { value =>
    LogLevel.logLevelMapping.get(value.toUpperCase) match {
      case Some(v) => Right(v)
      case None    => Left(Config.Error.InvalidData(Chunk.empty, s"Expected a log level, but found ${value}"))
    }
  }
```

?

Reply:
That looks fine.